### PR TITLE
cleanup of src directory

### LINKED ISSUE #4: Cleaning up the src directory
Clean up of the code in the `src` directory

### PR DESCRIPTION
**Pull request by FvanSteen**
fix #4

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -22,4 +12,4 @@ def perform_operation(num1, num2, operation):
     else:
         raise ValueError("Invalid operation. Please choose 'add' or 'subtract'.")
 
-    return result
\ No newline at end of file
+    return result
```
Comment (Eimolgon) [inline src/main.py:15]: Nice result bro

File: src/main.py
Comment (Eimolgon) [inline src/main.py:1]: Generic comment here

**Review by Eimolgon — COMMENTED** [review]
Generic comment again

File: src/main.py
Comment (FvanSteen) [reply src/main.py:1]: nice

**Review by Eimolgon — APPROVED** [review]

**Pull request merged**